### DISCUSSION TITLE
Usage with "products" instead of "projects"

### DISCUSSION THREAD

**Discussion by nolexio** [kimai/kimai#4844] [Questions & Answers]
Hello everyone!

First of all, I would like to thank you for the great software. I work in a company in the public sector. We have customers that I can of course cover with the software. Then we have "products" that can be used by several customers. As far as I can see, this is linked to the combination "customer", "project" in Kimai. However, projects take place on a further hierarchical level in our company, and I would rather place them in the "Activity" area. 

My question would therefore be whether there is a way to create "products" here and make them usable by several customers. Activities, or perhaps projects, would then be located on a further level.

Unfortunately, the fixed binding "customer" -> "project" with the binding to a customer makes the software unusable for us.

**Comment by kevinpapst** [comment]
> My question would therefore be whether there is a way to create "products" here and make them usable by several customers. Activities, or perhaps projects, would then be located on a further level.

You can test the Expenses plugin, which can work as products. I and several other customers use it for that purpose: 
https://www.kimai.org/store/expenses-bundle.html
